gitbook-docs/evm/endpoint.md

```diff
@@ -0,0 +1,257 @@
+# Proof Market Endpoint for Ethereum
+
+Proof Market EVM Endpoint is a solution that enhances the capabilities of [Proof Market](https://proof.market) by enabling direct proof ordering for Ethereum applications.
+This extension allows for trustless, provable computation composability for all EVM applications.
+The Ethereum Endpoint acts as a gateway interface facilitating communication between EVM applications and Proof Market.
+It enables several crucial operations:
+* Requesting zkProofs: EVM applications can directly request zero-knowledge proofs for specific computations.
+* Payment Facilitation: Payment for proof producers can be facilitated directly through Ethereum, streamlining the payment process.
```

Comment:
We should use a different word for the proof producer rewards. Asked folks in Slack.